src/tests/containerizer/cgroups2_tests.cpp

```diff
@@ -181,6 +194,55 @@ TEST_F(Cgroups2Test, ROOT_CGROUPS2_AssignProcesses)
 }
 
 
+TEST_F(Cgroups2Test, ROOT_CGROUPS2_Threads)
+{
+  const size_t NUM_THREADS = 5;
+  std::thread* runningThreads[NUM_THREADS];
+
+  Latch* latch = new Latch();
+
+  for (size_t i = 0; i < NUM_THREADS; ++i) {
+    runningThreads[i] = new std::thread([=]() {
+      // Wait until the main thread tells us to exit.
+      latch->await();
+    });
+  }
```

Comment:
per offline discussion, looks like this is carried over from the v1 tests, but this can leak during test failures

also, the test overall looks like it should take the same approach of forking and moving into a test cgroup?

Reply:
Updated to fork. Removed the `Latch` and made the threads `unique_ptr`s to have RAII.